podcasts/Bookmarks/List/BookmarkRow.swift

```diff
@@ -0,0 +1,104 @@
+import SwiftUI
+import PocketCastsUtils
+import PocketCastsDataModel
+
+struct BookmarkRow: View {
+    @EnvironmentObject var theme: Theme
+    @EnvironmentObject var viewModel: BookmarkListViewModel
+
+    private let bookmark: BookmarkManager.Bookmark
+
+    private let title: String
+    private let subtitle: String
+    private let playButton: String
+
+    @State private var higlighted = false
+
+    init(bookmark: BookmarkManager.Bookmark) {
```

Comment:
I'm not super sold on passing the bookmark here cause it feels like it should be a view model, but I'm not 100% sure what would be right.